src/components/Edit/ChangeSummary.vue

```diff
@@ -64,10 +64,6 @@ export default class ChangeSummary extends Mixins(
 
     return `${this.getBusinessNumber || '[Incorporation Number]'} B.C. Ltd.`
   }
-
-  onDeleteClicked (): void {
-    this.$root.$emit('delete-all')
```

Comment:
Is this event still used by another component?

Reply:
Not in this component, no. 
But Alteration Summary has the same event which is called in some unit testing for that Summary component. Perhaps that was caused it, since it wasn't a private method.